cilium-dbg/cmd/bgp_route_policy_get.go

```diff
@@ -68,7 +68,7 @@ func printBGPRoutePoliciesTable(policies []*models.BgpRoutePolicy) {
 	fmt.Fprintln(w, "VRouter\tPolicy Name\tType\tMatch Peers\tMatch Prefixes (Min..Max Len)\tRIB Action\tPath Actions")
 	for _, policy := range policies {
 		fmt.Fprintf(w, "%d\t", policy.RouterAsn)
-		fmt.Fprintf(w, "%s\t", trimString(policy.Name, 40))
+		fmt.Fprintf(w, "%s\t", trimString(policy.Name, 60))
```

Comment:
Is there a related constant for this length?

Reply:
Not really, as this policy name is generated dynamically by the agent based on BGP config in CRDs. This makes me think if the trim limit here actually makes sense - the point of the debug tool is to print the most accurate debug information, not to render visually clean output. Maybe I should remove the trim completely?

Reply:
Right, that's what I had in mind. If we include this output in sysdumps, we would want the full policy names.

Reply:
Another option, if you want alignment-calculation, is to use a TabWriter. See https://github.com/cilium/cilium/blob/main/cilium-dbg/cmd/bgp_peer_get.go#L84 for an example.

Reply:
ACK, removed the limit. The same TabWriter is used here in `printBGPRoutePoliciesTable` as well BTW :)